docs/getting-started/installing-the-us-forms-system-in-an-existing-application.md

```diff
@@ -0,0 +1,37 @@
+# Installing the US Forms System in an existing application
+
+If you're using an existing React application, you can install the US Forms System in that app.
+
+Before you begin, review "[Tools for getting started with the US Forms System](getting-started/tools-for-getting-started-with-the-us-forms-system)."
+
+### In this guide
+
+- [Install the library and dependencies](#install-the-library-and-dependencies)
+- [Create required files](#create-required-files)
+
+### Install the library and dependencies
+
+1. Install the library:
+```bash
+npm install --save https://github.com/usds/us-forms-system.git
+```
+
+2. Install peer dependencies:
+```bash
+npm install --save-dev @department-of-veterans-affairs/formation
```

Comment:
We no longer need formation, so you can remove this line!

Reply:
I was wondering about that one!